README.md

```diff
@@ -117,8 +117,40 @@ The body of the response MUST include a JSON object with a `prf` field. The valu
 { "prf": ["QmXiZ3sFXw811R8TrwaNeYvCF9Pv1nEmVpeEMEVpApzVhC", "bafkreidrgwjljxy6s7o5uvrifxnweffgi7chmye3pn6wyisv2n4b3uordi"] }
 ```
 
-# 4. Acknowledgments
+# 4 FAQ
+
+## 4.1 Why disallow duplicate headers?
+
+Duplicate headers are not handled consistently by all clients. Restricting to a single field is the simplest cross-client solution to this, as long as the spec is followed.
+
+## 4.2 Why not include the UCAN CIDs?
+
+By not including the CID in the header table, the recipient is forced to hash (and thus validate) the CIDs for each entry. If presented with a claimed CID as a table, implementers could ignore CID validation, breaking a core part of the proof chain security model. Hash functions are very fast on a couple kilobytes of data (the limit of most browsers in 8KB). While this strategy may be abused to force more hashing, the overhead is still very low.
```

Comment:
Is it worth referencing cache poisoning as a common category of attack mitigated by this design?

Reply:
Good idea — will do!